### DISCUSSION TITLE
Cannot Login VIa Okta with User Rights

### DISCUSSION THREAD

**Discussion by dustinkir** [requarks/wiki#6620] [Help / Questions] [answered]
We have Okta authentication working with users in the Administrators group in Wiki.js. If I put the same user in the "Users" group issues arise. 
Users will get through the Okta authentication, only to get sent back to the Unauthorized You cannot create the page screen. 
I've tried multiple browsers, private browser sessions, I've even enabled all available global permissions in the Users group to see if that helped - it did not. 
We need to be able to login as non admin users. This seems like a Wiki.js issue since the Okta component is working as expected. 
Any help will be greatly appreciated. 
I attached a screenshot of what we see after login as a member of the Users group. 
![Screenshot 2023-08-02 at 12 50 58 PM](https://github.com/requarks/wiki/assets/36971754/ce68f10f-6608-48af-a901-45f73572dd5a)

**Comment by NGPixel** [comment]
Does the Users group have page rules that enable read access to that page? Is the redirect after login pointing to a page that actually exists?

**Comment by dustinkir** [comment, marked as answer]
I had to create a default page in order to give the login process somewhere to land. That's all it took.
